frontend/src/components/ItemsList/StationsInfo.jsx

```diff
@@ -3,29 +3,39 @@ import "../../../style/itemsList.css";
 
 function StationInfo() {
   return (
-    <section className="stationsInfo">
-      <h2>Name Station</h2>
-      <div className="favorite" />
-      <img
-        className="BrandsStation"
-        src="https://via.placeholder.com/100"
-        alt="images"
-      />
-      <section className="stateStation">
-        <h2>6.8 Km</h2>
-        <div>
+    <div className="stationInfo">
+      <section className="nameStation">
```

Comment:
Your sections looks like figures because you have an img and some text each time. Take a look at figure html tag and how you can adapt it in your component. If you think it doesn't fit with what you want, use div instead of sections.

Reply:
still waiting for correction